lms/djangoapps/discussion/config/settings.py

```diff
@@ -0,0 +1,22 @@
+"""
+Discussion settings.
+"""
+from django.conf import settings
+
+# .. toggle_name: FEATURES['ENABLE_FORUM_DAILY_DIGEST']
+# .. toggle_implementation: DjangoSetting
+# .. toggle_default: True
+# .. toggle_description: Set to True to enable forum notification features.
+# .. toggle_category: discussion
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2020-03-09
+# .. toggle_expiration_date: None
+# .. toggle_tickets: None
+# .. toggle_status: supported
+# .. toggle_warnings: None
+ENABLE_FORUM_DAILY_DIGEST = 'enable_forum_daily_digest'
+
+
+def is_forum_daily_digest_enabled():
+    """Returns whether forum notification features should be visible"""
+    return settings.FEATURES.get('ENABLE_FORUM_DAILY_DIGEST')
```

Comment:
Django Setting best practices:
1. Should we always encapsulate checking the setting?  Could we do this in a standard way where we could better lint for toggle annotations?
2. Defaults: Should we always provide a default?  Should default of True have been added here in addition to, or in place of setting this to True in common.py?

Reply:
FWIW, I'd like to move us away from placing defaults in `common.py`.  In fact, the existence of a `common.py` module in a tangled monolith is far from ideal!  To move us to a "modular monolith", we need to have each module (Django app) as decoupled as possible - including the location and management of their settings and defaults.

Rather than inventing yet another way, can we build upon what was decided in OEP-45? See the [Django Settings section](https://github.com/edx/open-edx-proposals/pull/143/files#diff-9d5e701bc9f15777166f8459ecffb088R108).

Reply:
Sounds great.  Thanks.